charts/openebs/openebs-control-plane-pod-delete/experiment.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: litmuschaos.io/v1alpha1
+description:
+  message: |
+    Kill all openebs control plane pod and check if gets scheduled again
+kind: ChaosExperiment
+metadata:
+  labels:
+    litmuschaos.io/name: openebs
+  name: openebs-control-plane-pod-delete
+  version: 0.1.0
+spec:
+  definition:
+    scope: Namespaced
+    permissions:
+      - apiGroups:
+          - ""
+          - "litmuschaos.io"
+          - "batch"
+          - "apps"
+        resources:
+          - "pods"
+          - "deployments"
+          - "jobs"
+          - "configmaps"
+          - "chaosengines"
+          - "chaosexperiments"
+          - "chaosresults"
```

Comment:
are we now killing NDM pods? daemonsets

Reply:
But we are deleting the daemonsets's pods, not the daemonsets.